doc/source/whatsnew/v0.20.0.txt

```diff
@@ -51,6 +51,8 @@ Other enhancements
 
 - ``pd.read_excel`` now preserves sheet order when using ``sheetname=None`` (:issue:`9930`)
 
+- ``pd.io.gbq.read_gbq`` method now allows query configuration preferences (:issue:`14742`)
```

Comment:
@jorisvandenbossche about ```kwargs``` instead of ```configuration``` you can find there: https://github.com/pandas-dev/pandas/pull/14742#issuecomment-262965915
I think @jreback has better explanation why we use ```kwargs``` not ```configuration```